pom.xml

```diff
@@ -433,6 +434,21 @@
                 <version>${icu.version}</version>
             </dependency>
 
+            <!-- We need to fix our Scala versions as otherwise our depdendencies bring in
+            different versions via different routes
+            https://github.com/confluentinc/common/pull/260 -->
+            <dependency>
+                <groupId>org.scala-lang</groupId>
+                <artifactId>scala-library</artifactId>
+                <version>${scala.version}</version>
+            </dependency>
+
+            <dependency>
+                <groupId>org.scala-lang</groupId>
+                <artifactId>scala-reflect</artifactId>
+                <version>${scala.version}</version>
+            </dependency>
```

Comment:
Can we add this to `common` instead? KSQL and Schema Registry both seem to depend on a Scala library now (`mbknor-jackson-jsonschema`), so we should set this in the `dependencyManagement` section of `common`. And the scala version should be kept in sync with the one in Kafka.